src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -44,33 +46,8 @@ private static String listDepth(int depth){
         return String.join("", Collections.nCopies(depth+1, ":"));
     }
 
-    /**
-     * Convert some special tags to their asciidoc equivalents.
-     */
-    private static String stripDiv(String tagged){
-        //strip whitespace between tags
-        Matcher m = Pattern.compile("<p>((.|\n|\r)+?)</p>").matcher(tagged);
-        StringBuffer bufStr = new StringBuffer();
-        while(m.find()){
-            String rep = m.group();
-            rep = rep.replaceAll("\\n\\s+", "\n");
-	        m.appendReplacement(bufStr, Matcher.quoteReplacement(rep));
-        }
-        m.appendTail(bufStr);
-        tagged = bufStr.toString();
-
-        tagged = tagged.replaceAll("</?pre></?code>", "\n----\n")
-          .replaceAll("/?code></?pre>", "\n----\n") //otherwise hanging ``
-          .replaceAll("</?code>", "`")
-          .replaceAll("</?pre>", "\n----\n")
-          .replaceAll("</?div>", "")
-          .replaceAll("</?p>", "\n\n");
-        
```

Comment:
This was pretty much impossible to parse since there were so many alternative ways of classifying information.